addons/account/models/account_journal_dashboard.py

```diff
@@ -440,13 +440,16 @@ def action_create_new(self):
         else:
             ctx['default_move_type'] = 'entry'
             ctx['view_no_maturity'] = True
+        return ctx
+
+    def action_create_new(self):
         return {
             'name': _('Create invoice/bill'),
             'type': 'ir.actions.act_window',
             'view_mode': 'form',
             'res_model': 'account.move',
             'view_id': self.env.ref('account.view_move_form').id,
-            'context': ctx,
+            'context': self._get_move_action_context(),
```

Comment:
I would call it _get_move_action_context_from_journal because there are quite some methods playing with contexts in this file.  Otherwise lgtm  @lordkrandel